substrate/client/network/src/request_responses.rs

```diff
@@ -1734,4 +1846,4 @@ mod tests {
 			);
 		});
 	}
-}
+}
```

Comment:
Should be new line at EOF.

```suggestion
}

```